package/MDAnalysis/lib/distances.py

```diff
@@ -1523,3 +1526,45 @@ def apply_PBC(coords, box, backend="serial"):
         _run("triclinic_pbc", args=(coords, box), backend=backend)
 
     return coords
+
+
+def minimize_periodic_vector(reference_point, ctrpos, box, backend="serial"):
+    """Returns the periodic image of ctrpos which is closest to the reference_point
+
+    Parameters
+    ----------
+
+    reference_point : numpy.ndarray
+        Coordinate array of shape ``(3,)`` or ``(n, 3)`` containing the
+        position of reference_point (dtype is arbitrary, will be converted to
+        ``numpy.float32`` internally)
+    ctrpos : numpy.ndarray
```

Comment:
I do not like the name `ctrpos`. It is not very explicit, I'd rather have something like `center_position`. Also, I do not fully understand what it is about from the [arameter description.